helios-testing/src/main/java/com/spotify/helios/testing/HeliosDeploymentResource.java

```diff
@@ -97,12 +98,32 @@ public Boolean call() throws Exception {
           return null;
         }
 
-        if (!hosts.isEmpty()) {
-          log.info("Ensured that at least one agent is available in this HeliosDeployment, "
+        if (hosts.isEmpty()) {
+          log.debug("0 agents in {}, will retry", deployment);
+          return null;
+        }
+
+        log.info("Ensured that at least one agent has registered with ZooKeeper in this"
```

Comment:
This log message seems a little bit confusing, maybe just get rid of it. It's hard to know exactly what "agent has registered with ZooKeeper" means